bpfman/src/serve.rs

```diff
@@ -117,12 +126,25 @@ pub async fn serve(
     Ok(())
 }
 
-pub(crate) async fn shutdown_handler() {
+pub(crate) async fn shutdown_handler(use_activity_timer: bool) {
+    const TIMEOUT: std::time::Duration = std::time::Duration::from_secs(15);
```

Comment:
I think you can use https://blog.logrocket.com/rust-lazy-static-pattern/ for this since then it won't actually get initialized unless it's used